sprint_boundary_iterator.rb

```diff
@@ -19,6 +19,12 @@ def each
     end
   end
 
+  def self.start_range
+    self.new.each do | number, range |
+      return range
+    end
```

Comment:
```suggestion
    self.new.first.last
```

Reply:
(or `self.new.first[1]`, whichever you prefer.

Reply:
Implemented and tested locally.